source/libs/page-detect.ts

```diff
@@ -75,7 +75,7 @@ export const isReleasesOrTags = (): boolean => /^(releases|tags)/.test(getRepoPa
 export const isEditingFile = (): boolean => /^edit/.test(getRepoPath()!);
```

Comment:
```suggestion
export const isEditingFile = (): boolean => getRepoPath()?.startsWith('edit');
```

Applies to many others in this file.

Reply:
<img width="578" alt="undefined" src="https://user-images.githubusercontent.com/1402241/68443235-ecd9a500-0205-11ea-95d2-f8c50531950b.png">

This results in `boolean | undefined`. I'd probably have to change it to:

```suggestion
export const isEditingFile = (): boolean => getRepoPath()?.startsWith('edit') || false;
```

Worth it?